### ISSUE TITLE
Hide or disable the focus ring in Slide

### ISSUE THREAD

**Issue by abrevinci** [express-labs/pure-react-carousel#404]
**Describe the feature you'd like:**

I just spent a lot of time trying to get rid of the focus ring in the Slide component. I believe this library is missing the capability to customize this part of the rendering. 

**Suggested implementation:**

I'd suggest exposing the ability to set classNames for it like can be done for many of the other elements. That way, it can be hidden or styled to anyones content.

**Describe alternatives you've considered:**

I could override the css class assigned to the focus ring div, but I am using tailwind and overriding global styles feels really hacky. It feels a little bit like this is simple an oversight that it has not been exposed as a classname prop, especially since this library is supposed to be "lookless".

**Teachability, Documentation, Adoption, Migration Strategy:**

The slider component would get an additional focusClassName prop that defaults to null and is not required. It is appended to the className of the focus ring element.

I'd be happy to submit a PR for this as it is a very simple change.

**Comment by abrevinci** [comment]
I cannot get this project to build on my computer. Something fishy with my environment probably. I have implemented this but am not able to test them due to this. Let me know if I should still open a PR for someone else to test it who doesn't have issues. 

The issues I am facing are related to npm, node-sass, package.json scripts that are invalid in both git bash and cmd (I'm on windows), and just in general an inexperience with the build tools used. I have never used rollup for instance and can't get it to work.

**Comment by abrevinci** [comment]
I ended up working around this by simply assigning focus-visible styling to the slide element itself and hiding the focus ring added by pure-react-carousel using global css:

.carousel__slide-focus-ring {
  display: none !important;
  outline-width: 0 !important;
}

It completely beats me why the focus ring is in a separate element instead of just giving some default outline to the slide element on focus-visible like this.
